apps/cowswap-frontend/src/common/containers/LimitOrdersPromoBanner/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useAtom } from 'jotai'
+import { useCallback } from 'react'
+
+import { useTradeNavigate } from 'modules/trade'
+
+import { Routes } from '../../constants/routes'
+import { LimitOrdersPromoBanner as LimitOrdersPromoBannerPure } from '../../pure/LimitOrdersPromoBanner'
+import { limitOrdersPromoDismissedAtom } from '../../state/limitOrdersPromoAtom'
+
+interface LimitOrdersPromoBannerProps {
+  isLimitOrdersTab?: boolean
+}
+
+export function LimitOrdersPromoBanner({ isLimitOrdersTab = false }: LimitOrdersPromoBannerProps) {
+  const [isDismissed, setIsDismissed] = useAtom(limitOrdersPromoDismissedAtom)
+  const tradeNavigate = useTradeNavigate()
+
+  const handleDismiss = useCallback(() => {
```

Comment:
```
const handleDismiss = () => setIsDismissed(true)
```

No need to use `useCallback` since the last React update

Reply:
This is what I read on it https://github.com/facebook/react/issues/31913#issuecomment-2568316803

But I will remove it.